src/libs/actions/Report.js

```diff
@@ -121,9 +121,16 @@ function getUnreadActionCount(report) {
  * @param {Object} report
  * @return {String[]}
  */
-function getParticipantEmailsFromReport({sharedReportList, reportNameValuePairs}) {
+function getParticipantEmailsFromReport({sharedReportList, reportNameValuePairs, ownerEmail}) {
     const emailArray = _.map(sharedReportList, participant => participant.email);
-    return (ReportUtils.isChatRoom(reportNameValuePairs) || ReportUtils.isPolicyExpenseChat(reportNameValuePairs)) ? emailArray : _.without(emailArray, currentUserEmail);
+    if (ReportUtils.isChatRoom(reportNameValuePairs)) {
+        return emailArray;
+    }
+    if (ReportUtils.isPolicyExpenseChat(reportNameValuePairs)) {
+        // The owner of the policyExpenseChat isn't in the sharedReportsList so they need to be explicitely included.
+        return [ownerEmail, ...emailArray];
+    }
+    return _.without(emailArray, currentUserEmail);
```

Comment:
NAB, I don't know how long it has been this way. But it would be nice to call out why the `currentUserEmail` is not being included here. Maybe something like:

>The current user always appears in the `sharedReportsList` for regular chat reports, but get excluded from the `participants` array because... 

And I was going to finish that comment, but I can't quite remember the answer 😅

Reply:
Ah I believe it's because typically chatReports are DM's and groupDMs and you participation is implied if you're in a DM. I can put that in a comment somehow.

Reply:
Looks great thanks.